src/libstd/collections/hash/table.rs

```diff
@@ -899,10 +886,8 @@ impl<K, V> RawTable<K, V> {
     /// Returns an iterator that copies out each entry. Used while the table
     /// is being dropped.
     unsafe fn rev_move_buckets(&mut self) -> RevMoveBuckets<K, V> {
-        let raw_bucket = self.first_bucket_raw();
         RevMoveBuckets {
-            raw: raw_bucket.offset(self.capacity as isize),
-            hashes_end: raw_bucket.hash,
+            raw: self.raw_bucket_at(self.capacity()),
```

Comment:
This is intentionally set to one past the end for the reverse style iterator. I will add a comment line on that.